runtime/common/RestClient.cpp

```diff
@@ -48,6 +49,38 @@ std::string decompress_gzip(const std::string &data) {
   return decompressed;
 }
 
+RestClient::RestClient() : sslOptions(std::make_unique<cpr::SslOptions>()) {
+  auto caInfo = [&]() -> std::string {
+    if (auto *curlCABundleStr = getenv("CURL_CA_BUNDLE")) {
+      if (std::filesystem::exists(curlCABundleStr))
+        return curlCABundleStr;
+      else
+        cudaq::info(
+            "{} does not exist. Will fall back on CUDAQ installed certs",
+            curlCABundleStr);
+    }
+    std::filesystem::path cudaqLibPath{cudaq::getCUDAQLibraryPath()};
+    auto certPath = cudaqLibPath.parent_path().parent_path() / "cacert.pem";
```

Comment:
@bettinaheim please double check me on this path. It would be map to `/opt/nvidia/cudaq/cacerts.pem` once everything is installed.

Reply:
> @bettinaheim please double check me on this path. It would map to `/opt/nvidia/cudaq/cacerts.pem` once everything is installed (or the top level of the Python wheel)

I checked this in the `latest-base` image, which includes the change to include that file, and it is indeed in that location.